dipy/utils/tests/test_parallel.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import numpy.testing as npt
+
+import dipy.utils.parallel as para
+
+
+def power_it(num, n=2):
+    # We define a function of the right form for parallelization
+    return num ** n
+
+def test_parfor():
+    engines = ["serial"]
+    if para.has_dask:
+        engines.append("dask")
+    if para.has_joblib:
+        engines.append("joblib")
+    if para.has_ray:
+        engines.append("ray")
+
+    my_array = np.arange(100).reshape(10, 10)
+    i, j = np.random.randint(0, 9, 2)
+    my_list = list(my_array.ravel())
+    for engine in engines:
+        for backend in ["threading", "multiprocessing"]:
+            npt.assert_equal(para.parfor(power_it,
```

Comment:
Why do you test only 1 pixel and not the whole array?

Reply:
I am not sure 😄 

Changed in 807ac1f1d to test the whole array.